.github/workflows/tests.yml

```diff
@@ -79,19 +79,6 @@ jobs:
           tags: |
             permitio/opal-client:test
 
-      - name: Build client cedar
```

Comment:
Why is this removed?

Reply:
@danyi1212 
Why is it needed? The images are being built here without being pushed, just to be used for tests in the next step.
The cedar image is not being tested, so why build it? It takes a lot of time... 
(I know I've approved that PR - but just noticed later)

Reply:
Doesn't it test the docker build itself? If its unused, we can drop it

Reply:
That wasn't the goal to begin with. But I guess it does?
It's not like the build fails with compilation errors or something :)